controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/integration/dns/Record.java

```diff
@@ -57,22 +51,30 @@ public RecordName name() {
 
     @Override
     public String toString() {
-        return String.format("%s: %s %s -> %s", id, type, name, data);
+        return String.format("%s %s -> %s", type, name, data);
     }
 
     @Override
     public boolean equals(Object o) {
         if (this == o) return true;
         if (o == null || getClass() != o.getClass()) return false;
         Record record = (Record) o;
-        return Objects.equals(id, record.id) &&
-               type == record.type &&
-               Objects.equals(name, record.name) &&
-               Objects.equals(data, record.data);
+        return type == record.type &&
+               name.equals(record.name) &&
+               data.equals(record.data);
     }
 
     @Override
     public int hashCode() {
-        return Objects.hash(id, type, name, data);
+        return Objects.hash(type, name, data);
+    }
+
+    @Override
+    public int compareTo(Record that) {
+        return Comparator.comparing(Record::type)
```

Comment:
Consider making the comparator static to avoid creating one for each comparisom :)